pkg/resources/view_acceptance_test.go

```diff
@@ -89,24 +117,32 @@ func TestAcc_View_basic(t *testing.T) {
 		Steps: []resource.TestStep{
 			// without optionals
 			{
-				Config: accconfig.FromModel(t, viewModelWithDependency),
-				Check: assert.AssertThat(t, resourceassert.ViewResource(t, "snowflake_view.test").
-					HasNameString(id.Name()).
-					HasStatementString(statement).
-					HasDatabaseString(id.DatabaseName()).
-					HasSchemaString(id.SchemaName())),
+				ConfigDirectory: acc.ConfigurationDirectory("TestAcc_View/basic"),
+				ConfigVariables: basicView(statement),
+				Check: assert.AssertThat(t,
+					resourceassert.ViewResource(t, "snowflake_view.test").
+						HasNameString(id.Name()).
+						HasStatementString(statement).
+						HasDatabaseString(id.DatabaseName()).
+						HasSchemaString(id.SchemaName()),
+					assert.Check(resource.TestCheckResourceAttr("snowflake_view.test", "column.#", "2")),
+				),
 			},
 			// import - without optionals
 			{
-				Config:       accconfig.FromModel(t, viewModel),
-				ResourceName: "snowflake_view.test",
-				ImportState:  true,
-				ImportStateCheck: assert.AssertThatImport(t, assert.CheckImport(importchecks.TestCheckResourceAttrInstanceState(resourceId, "name", id.Name())),
+				ConfigDirectory: acc.ConfigurationDirectory("TestAcc_View/basic"),
+				ConfigVariables: basicView(statement),
+				ResourceName:    "snowflake_view.test",
+				ImportState:     true,
+				ImportStateCheck: assert.AssertThatImport(t,
+					assert.CheckImport(importchecks.TestCheckResourceAttrInstanceState(resourceId, "name", id.Name())),
```

Comment:
nit: this check is also below: `HasNameString(id.Name())`

Reply:
True, removed this one.